README.linux.md

```diff
@@ -57,31 +57,21 @@
     ./README.CentOS.bash
     ```
 
-## For Ubuntu:
-
-- Install Dependencies
-  When you run the README.ubuntu.bash script for dependencies, you will be asked to configure realm for kerberos.
-  You can enter any realm, since this is just for testing, and during testing, it will reconfigure a local server/client.
-  If you want to skip this manual configuration, use:
-  `export DEBIAN_FRONTEND=noninteractive`
+## For Ubuntu (versions 20.04 or newer):
 
+- Install dependencies using README.ubuntu.bash script:
```

Comment:
was kerberos part omitted intentionally?

Reply:
Since I've removed `krb5-kdc` and `krb5-admin-server` from the install dependencies, there is no configuration action required during the installation.

Reply:
Kerberos is referenced in many parts of source code and documentation. Although it's possible to have some build and demo cluster without it, it might be required for more complex setup. I suggest to get it back and mark as optional.

Reply:
Ok, sorry, I just noticed a new comment here but have already submitted a request. I'll return Kerberos now and notify you

Reply:
Done.